index.html

```diff
@@ -356,7 +356,7 @@ <h1 class="title">Promise.prototype.finally</h1>
 		</emu-alg>
 	</emu-clause>
 
-	<!-- es6num="25.4.1.8" -->
+	<!-- es6num="25.4.4.5" -->
```

Comment:
Unless I copy-paste-errored, 25.4.1.8 is what it was when I first copied the spec steps for `Promise.resolve` and started writing this PR. Hopefully it's unchanged and I just messed it up :-)